src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/opentelemetry/otel-adot-log-ingestion-serverless.mdx

```diff
@@ -0,0 +1,191 @@
+---
+title: OTEL log ingestion solution for ADOT lambda layer collector
+metaDescription: How to instrument your serverless applications on AWS Lambda with ADOT solution in New Relic.
+freshnessValidatedDate: never
+---
+You can now monitor AWS serverless application that sends OTEL log events data from CloudWatch Logs to the New Relic UI.
+You can configure the ADOT lambda layer collector to successfully forward OTEL application logs with all necessary service and application context. 
```

Comment:
It's a subtle thing but I think we can make a slight tweak to make this read as a bit friendlier:

"You can configure the ADOT Lambda layer collector to successfully forward your OTEL application logs with all of your service and application context."